manifests/v1alpha3/katib-controller/katib-config.yaml

```diff
@@ -34,6 +34,7 @@ data:
         "image": "gcr.io/kubeflow-images-public/katib/v1alpha3/suggestion-hyperopt"
       },
       "nasrl": {
-        "image": "gcr.io/kubeflow-images-public/katib/v1alpha3/suggestion-nasrl"
+        "image": "gcr.io/kubeflow-images-public/katib/v1alpha3/suggestion-nasrl",
+        "memLimit": "200Mi"
```

Comment:
I prefer that here keep same style as k8s resources structure like below
```
   image: gcr.io/kubeflow-images-public/katib/v1alpha3/suggestion-nasrl
   resources:
      requests:
        memory: "64Mi"
        cpu: "250m"
      limits:
        memory: "128Mi"
        cpu: "500m"
```

Reply:
Agree.

Reply:
I changed structure for configMap.